docs/src/main/paradox/http-server.md

```diff
@@ -50,38 +66,46 @@ Note the following building blocks from the snippet:
 
 Next, the example defines how to retrieve and remove a user. In this case, the URI must include the user's id in the form: `/users/$ID`. See if you can identify the code that handles that in the following snippet. This part of the route includes logic for both the GET and the DELETE methods.
 
-@@snip [QuickstartServer.scala]($g8src$/scala/com/lightbend/akka/http/sample/QuickstartServer.scala) { #users-get-delete }
+ą) { #users-get-delete }
 
 This part of the `Route` contains the following:
 
 **Generic functionality**
 
 * `pathPrefix("users")` : the path that is used to match the incoming request against.
+* `concat`: concatenates two or more route alternatives. Routes are attempted one after another. If a route rejects a request, the next route in the chain is attempted. This continues until a route in the chain produces a response. 
```

Comment:
Aren't we repeating the same things again here? I see it's a shorter version, but maybe we could replace the explanation with something like "see above"